GLTFSDK/Source/GLTFResourceReader.cpp

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <GLTFSDK/GLTFResourceReader.h>
+
+using namespace Microsoft::glTF;
+
+namespace
+{
+	// Conversions of normalized component types to/from floats are explicitly defined in the 2.0 spec

```

Comment:
These ComponentToFloat functions are now defined twice (here and in AnimationUtils.h). They should only exist once.

Reply:
I specifically kept the AnimationUtils.h versions because it wasn't clear to me if they are part of the public interface or not. I can remove them if they aren't?

Reply:
For now I renamed the copies of ComponentToFloat in the internal reader implementation to DecodeToFloat to avoid confusion.

Reply:
I don't think they were intended to be part of the public interface but they are called from two locations in AnimationUtilsTests.cpp. I think they should either be moved to ResourceReaderUtils.h or, if the code has to be duplicated, then the 'old' ComponentToFloat functions should be moved into AnimationUtilsTests.cpp